sdk/packages/core/src/contracts/parseOpenEvent.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  type DecodeEventLogReturnType,
+  type Log,
+  decodeEventLog,
+  parseEventLogs,
+} from 'viem'
+import { inboxABI } from '../constants/abis.js'
+import { ParseOpenEventError } from '../errors/base.js'
+
+export type ParseOpenEventReturn = DecodeEventLogReturnType<
+  typeof inboxABI,
+  'Open'
+>['args']['resolvedOrder']
+
+export function parseOpenEvent(logs: Log[]): ParseOpenEventReturn {
```

Comment:
Can we just have the return a `ResolvedOder`

Reply:
👍 updated